x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -1009,5 +1009,67 @@ export const registerEnterpriseSearchIntegrations = (
       shipper: 'enterprise_search',
       isBeta: false,
     });
+
+    customIntegrations.registerCustomIntegration({
+      id: 'notion',
+      title: i18n.translate('xpack.enterpriseSearch.integrations.connectors.notionTitle', {
+        defaultMessage: 'Notion',
+      }),
+      description: i18n.translate(
+        'xpack.enterpriseSearch.integrations.connectors.notionDescription',
+        {
+          defaultMessage: 'Search over your content on Notion.',
+        }
+      ),
+      categories: [
+        'enterprise_search',
+        'elastic_stack',
+        'custom',
+        'connector',
+        'connector_client',
+        'notion',
```

Comment:
As in the other PR, 'notion' and 'redis' should not be categories :)

Reply:
Removed in https://github.com/elastic/kibana/pull/177306/commits/905fa7d8f5b3dd27e78d6d7b0e7597e33b3798e8